uwblueprint/jump-math
# feat: admin & teacher login pages

## Notion ticket link

[Admin Login Page](https://www.notion.so/uwblueprintexecs/Admin-Login-Page-775b7d517b544ef1a6e5ba48a613c372)
[Teacher Login Page](https://www.notion.so/uwblueprintexecs/Teacher-Login-Page-e654146e2bfd4cf2b961d49dfecb2461)



## Implementation description
* Moved all image imports to `assets` folder and removed image files
*  Created `AuthTemplate` as a wrapper to all the authentication pages (that use this two column format) and refactored existing pages to use this wrapper
* Moved `FormError` into outer folder so we can use it across the authentication pages
* Updated `Login` component to render the correct pages according to design
* Updated `Landing` to navigate to the the correct `Login` pages
* Updated `routes` accordingly



## Steps to test
1. Test Admin login flow
2. Test Teacher login flow
3. Both should resize properly as well

**Admin Login**
<img width="1920" alt="Screen Shot 2023-01-05 at 4 13 42 PM" src="https://user-images.githubusercontent.com/41273486/210881156-8541b216-fd35-4bd0-b9a1-0aba94c25381.png">

**Teacher Login**
<img width="1920" alt="Screen Shot 2023-01-05 at 4 13 34 PM" src="https://user-images.githubusercontent.com/41273486/210881142-d27fede4-3b92-4cad-b66f-8804da02b8b6.png">
